2_Collections.py

```diff
@@ -0,0 +1,42 @@
+import random  # Import the random module to generate random numbers
+import string  # Import the string module to access string constants like lowercase letters
+
+dicts_list = []  # Initialize an empty list to hold the generated dictionaries
+num_dicts = random.randint(2, 10)  # Generate a random number of dictionaries (between 2 and 10)
+
+for i in range(num_dicts):  # Loop to create the specified number of dictionaries
+    num_keys = random.randint(3, 5)  # Generate a random number of keys for each dictionary (between 3 and 5)
+    new_dict = {random.choice(string.ascii_lowercase): random.randint(0, 100) for _ in
+                range(num_keys)}  # Create a new dictionary with random keys (letters) and random values (0-100)
+    sorted_dict = dict(sorted(new_dict.items()))  # Sort the dictionary by its keys and convert it back to a dictionary
+    dicts_list.append(sorted_dict)  # Append the sorted dictionary to the list
```

Comment:
Read also about list comprehensions + dict comprehensions. You will see such code in other projects.
https://realpython.com/list-comprehension-python/